### DISCUSSION TITLE
DateInput "onPick" event

### DISCUSSION THREAD

**Discussion by dev-banana** [mantinedev/mantine#6761] [Q&A] [answered]
I'm using the “DateInput” component: 
I'd like a way to differentiate the change of value because I typed in the Input field, from the change of value after clicking on a day in the calendar.
I'm looking for an event such as “onPick”... Has anyone ever had this kind of situation and solved it?

**Comment by rtivital** [comment, marked as answer]
`<DateInput getDayProps={date => ({ onClick: console.log(date) })} />`

**Comment by dev-banana** [comment]
Thanks for your reply, it was exactly what I was looking for :)

However, the “onClick” event is never triggered, whereas it works with the “onClickCapture” event... Any idea why?

`<DateInput getDayProps={date => ({
     onClickCapture: () => console.log("onClickCapture", date),
     onClick: () => console.log("onClick", date),
 })} />`

**Reply by rtivital** [reply]
I don't know, I'll have a look and check why it does not work. Meanwhile you can use onClickCapture.
